### ISSUE TITLE
Add "hello world" scene

### ISSUE THREAD

**Issue by StevenClontz** [StevenClontz/manim-workshop#1]
We want a scene that renders in seconds for users to confirm their Codespace environment is set up correctly.

**Comment by StevenClontz** [comment]
closed by @TienChih in [d69ab1a](https://github.com/StevenClontz/manim-workshop/commit/d69ab1a96dffacebd2a0b3aaa38366a4401507e1)

**Comment by StevenClontz** [comment]
Ran in just a few seconds:

```
manimuser@codespaces-16e056:/workspaces/manim-workshop$ manim render -ql main.py HelloWorld
[02/23/24 15:01:40] WARNING  Missing Plugins: {''}                                                                                               __init__.py:17
Manim Community v0.18.0

                    INFO     Writing \text{Hello World!} to media/Tex/6edf49a892ed6f3f.tex                                              tex_file_writing.py:109
[02/23/24 15:01:43] INFO     Writing \text{Welcome to MAA SE 2024!} to media/Tex/03d8104208af639f.tex                                   tex_file_writing.py:109
[02/23/24 15:01:46] INFO     Animation 0 : Partial movie file written in                                                               scene_file_writer.py:527
                             '/workspaces/manim-workshop/media/videos/main/480p15/partial_movie_files/HelloWorld/1185818338_643817416_                         
                             223132457.mp4'                                                                                                                    
[02/23/24 15:01:47] INFO     Animation 1 : Partial movie file written in                                                               scene_file_writer.py:527
                             '/workspaces/manim-workshop/media/videos/main/480p15/partial_movie_files/HelloWorld/624642324_255102487_1                         
                             246481580.mp4'                                                                                                                    
                    INFO     Animation 2 : Partial movie file written in                                                               scene_file_writer.py:527
                             '/workspaces/manim-workshop/media/videos/main/480p15/partial_movie_files/HelloWorld/624642324_774349388_5                         
                             99705372.mp4'                                                                                                                     
                    INFO     Animation 3 : Partial movie file written in                                                               scene_file_writer.py:527
                             '/workspaces/manim-workshop/media/videos/main/480p15/partial_movie_files/HelloWorld/624642324_2475379470_                         
                             1644797632.mp4'                                                                                                                   
[02/23/24 15:01:48] INFO     Animation 4 : Partial movie file written in                                                               scene_file_writer.py:527
                             '/workspaces/manim-workshop/media/videos/main/480p15/partial_movie_files/HelloWorld/624642324_132941649_1                         
                             028800789.mp4'                                                                                                                    
                    INFO     Animation 5 : Partial movie file written in                                                               scene_file_writer.py:527
                             '/workspaces/manim-workshop/media/videos/main/480p15/partial_movie_files/HelloWorld/624642324_2475379470_                         
                             2516485146.mp4'                                                                                                                   
[02/23/24 15:01:49] INFO     Animation 6 : Partial movie file written in                                                               scene_file_writer.py:527
                             '/workspaces/manim-workshop/media/videos/main/480p15/partial_movie_files/HelloWorld/624642324_1360923468_                         
                             741294118.mp4'                                                                                                                    
                    INFO     Animation 7 : Partial movie file written in                                                               scene_file_writer.py:527
                             '/workspaces/manim-workshop/media/videos/main/480p15/partial_movie_files/HelloWorld/624642324_255102487_2                         
                             977803849.mp4'                                                                                                                    
                    INFO     Combining to Movie file.                                                                                  scene_file_writer.py:617
                    INFO                                                                                                               scene_file_writer.py:737
                             File ready at '/workspaces/manim-workshop/media/videos/main/480p15/HelloWorld.mp4'                                                
                                                                                                                                                               
                    INFO     Rendered HelloWorld                                                                                                   scene.py:247
                             Played 8 animations   
```
